builtin/settingtypes.txt

```diff
@@ -658,8 +658,9 @@ texture_path (Texture path) path
 #    The rendering back-end for Irrlicht.
 #    A restart is required after changing this.
 #    Note: On Android, stick with OGLES1 if unsure! App may fail to start otherwise.
-#    On other platforms, OpenGL is recommended, and it’s the only driver with
-#    shader support currently.
+#    On other platforms, OpenGL is recommended.
+#    Shaders are supported by OpenGL (desktop only) and OGLES2 (experimental,
+#    Android only)
```

Comment:
```suggestion
#    Shaders are supported by OpenGL (desktop only) and OGLES2 (experimental)
```

Reply:
Does that mean GLES2 is supported on desktop?